conversion enu/flu

BenjaminHug8 opened the pull request:
This PR prepare the conversion enu/flu that will be introduce in `Webots R2022a`.

joachimhgg reviewed (changes requested):
I tried to convert the kuka following this tutorial: https://github.com/cyberbotics/urdf2webots/blob/master/docs/tutorial.md but I got this result:
![image](https://user-images.githubusercontent.com/48200998/141949140-9dae70ca-df6a-4b3e-9184-b5c0b0201e47.png)

We also need to update the tutorial.

BenjaminHug8 commented:
> I tried to convert the kuka following this tutorial: https://github.com/cyberbotics/urdf2webots/blob/master/docs/tutorial.md but I got this result

It does not work in `master` either...
I am going to check if that is coming from the new linking of the meshes.

BenjaminHug8 commented:
Ok it seems that this kuka arm is missing some rotation in the URDF thus also the XACRO compared to the test on CI with motoman.

BenjaminHug8 commented:
Up to version 1.0.10 it works correctly.

The base missing is not surprising as it is defined with an indexed face set that I removed when linking the meshes.

But I have to solve the orientation of each part and the textures missing, thanks for spotting this!

BenjaminHug8 commented:
> I have to solve the orientation of each part and the textures missing, thanks for spotting this!

This will be done in other PR.

BenjaminHug8 commented:
> I tried to convert the kuka following this tutorial: https://github.com/cyberbotics/urdf2webots/blob/master/docs/tutorial.md but I got this result: ![image](https://user-images.githubusercontent.com/48200998/141949140-9dae70ca-df6a-4b3e-9184-b5c0b0201e47.png)

This has been solved in another PR (the mesh linking was not working properly for `.dae` files).

joachimhgg commented on urdf2webots/importer.py:
```diff
@@ -208,7 +208,7 @@ def convert2urdf(inFile, outFile=None, normal=False, boxCollision=False,
+    optParser.add_option('--rotation', dest='initRotation', default='0 0 1 0',
                          help='Set the rotation field of your PROTO file.')
     optParser.add_option('--init-pos', dest='initPos', default=None,
                          help='Set the initial positions of your robot joints. Example: --init-pos="[1.2, 0.5, -1.5]" would '
```
maybe the example of the initial-pos needs to be adapt

BenjaminHug8 replied:
This is an option, it is not related to one example in particular.

joachimhgg reviewed (changes requested):
The images need to be update. (I don't have the meshes folder anymore, etc)
![image](https://user-images.githubusercontent.com/48200998/142843781-b372e97c-f784-4002-8c4f-f1e79aa76b20.png)
![image](https://user-images.githubusercontent.com/48200998/142844088-d3fb7a05-6cae-4275-a559-3e20aa1b2ac1.png)

BenjaminHug8 commented:
Images changes.

joachimhgg reviewed (approved):
Thanks!

BenjaminHug8 commented:
@omichel I ask you to review once again as I prepared the `1.0.13` release and removed the python 2 test.

I removed some white spaces at the end of some lines added during the creation of the PROTO.
There is also extra space and comma after the last item of some list but I kept them as it would take a bit more place in the code to only to polish a huge list of point. In my opinion it is not worth to do it but let me know if you think it is a good idea to remove these extra characters.

omichel reviewed (approved):
Looks good to me. Thank you.
